Apps/Database/src/Models/UserProfile.cs

```diff
@@ -72,5 +73,10 @@ public class UserProfile : AuditableEntity
         /// </summary>
         [MaxLength(44)]
         public string? EncryptionKey { get; set; }
+
+        /// <summary>
+        /// Gets the list containing all the communication emails related to this user profile.
+        /// </summary>
+        public virtual List<CommunicationEmail> CommunicationEmails { get; } = null!;
```

Comment:
We did this a lot early on but we've been leaning against it lately and instead have been creating empty lists.  We should discuss during grooming.

Reply:
Oh I didn't know that. One benefit of this is ability to include the list or a filtered list of child records in the LINQ query (i.e. .Include(...)

Reply:
updated to "public virtual List<CommunicationEmail> CommunicationEmails { get; } = new List<CommunicationEmail>();"